src/slack.ts

```diff
@@ -1,7 +1,15 @@
 // -----------------------------------------------------------------------------
 // Bot / Slack related helpers
 // -----------------------------------------------------------------------------
-import { RTMClient } from '@slack/rtm-api'
+import { App } from '@slack/bolt'
+
+// Load environment variables; don't delete this
+// This is particularly for Events API stuff
+import dotenv from 'dotenv'
```

Comment:
THis shouldn't be in the live / prod version, if it is, it should be in a different file, like main.